### PR TITLE
Added roadmap link

### LINKED ISSUE #181: Add roadmap link to the documentation section in readme
## Is your feature request related to a problem? Please describe

We can get the idea of the features and project status by  the project roadmap, hence a link to the same should be present in the readme.

## Describe the solution you'd like

Add the roadmap link under documentation section in Readme.

### PR DESCRIPTION
**Pull request by Samridhi-98**
### Description

Fixes #181 

### Type of Change:

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@isabelcosta  sorry for the inconvenience I have opened a new PR as previous branch got deleted locally by mistake, please review this PR.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ Anitab-Forms is an application that simplifies the processing and selection proc
 
 Documentation for the project is hosted [here](https://anitab-org.github.io/anitab-forms-web/). We use `Docusaurus` for maintaining the documentation of the project.
 
+## Roadmap
+
+View project Roadmap [here](https://anitab-org.github.io/anitab-forms-web/docs/).
```
Comment (Aaishpra) [inline README.md]:
```suggestion
View project Roadmap [here](https://github.com/orgs/anitab-org/projects/7#card-58848156)
```

Comment (Aaishpra) [inline README.md]: The link to roadmap was wrong one, so please add this one here. Thanks

Comment (Samridhi-98) [reply README.md]: Done :)

**Review by Aaishpra — APPROVED** [review]
Lgtm 👍🏻 thanks for this @Samridhi-98

**Review by isabelcosta — APPROVED** [review]
Great work @Samridhi-98 !

**Comment by isabelcosta** [comment]
Thank you for reviewing this @Aaishpra !

**Pull request merged**